src/NServiceBus.Core/Licensing/LicenseManager.cs

```diff
@@ -71,7 +71,7 @@ static License GetTrialLicense()
             }
             else
             {
-                var message = $"Trial for Particular Service Platform is still active, trial expires on {trialLicense.ExpirationDate.Value.ToLocalTime().ToShortDateString()}.";
+                var message = $"Trial for the Particular Service Platform has been active since {trialStartDate.ToLocalTime().ToShortDateString()}.";
```

Comment:
@ramonsmits we deliberately avoided that, since it re-inforces the feeling of "OMG, I've only got 5 days to trial this thing!"